src/vision/src/hsv_filter_node.cpp

```diff
@@ -7,12 +7,9 @@
 #include <HSVFilterNode.h>
 
 int main(int argc, char** argv) {
-    // Setup your ROS node
-    std::string node_name = "filter";
-
-    // Create an instance of your class
+    //     Create an instance of your class
+    std::string node_name = "hsv_filter_node";
     HSVFilterNode filter(argc, argv, node_name);
-
-    // Once the node stops, return 0
+    //     Once the node stops, return 0
     return 0;
 }
```

Comment:
nit github complaining about end-of-file

Reply:
Done

Reply:
Still complaining?